src/utils/node.ts

```diff
@@ -73,3 +73,8 @@ export const startLiskCore = async (
 
 	return execAsync(`lisk-core start--network ${network} --api - ipc--log info`);
 };
+
+export const resolveAbsolutePath = (path: string) => {
```

Comment:
Shouldn't this go to the fs utils file instead? Let's create one, if it doesn't exist.